x-pack/legacy/plugins/cross_cluster_replication/public/app/components/follower_index_form/follower_index_form.js

```diff
@@ -30,6 +29,7 @@ import {
   EuiTitle,
 } from '@elastic/eui';
 
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from '../../../../../../../../src/plugins/es_ui_shared/public';
```

Comment:
It would be better to namespace our export.

```js
import { indices } from '../../../../../../../../src/plugins/es_ui_shared/public;

const { INDEX_ILLEGAL_CHARACTERS_VISIBLE } = indices;
```
This will avoid naming collisions and better organization of all our exports.